index.html

```diff
@@ -242,6 +243,7 @@ <h3>
             <li>Storage and Sharing of Verifiable Credentials</li>
             <li>Privacy for Verifiable Credentials</li>
             <li>An API for Verifiable Credential Exchange</li>
+            <li>Extensions for binding multilingual strings to GNU gettext-like localization files</li>
```

Comment:
Only a minor remark: it may be helpful to turn the "GNU gettext-like localization files" into an active link to a specification.

Reply:
Updated. thanks!